proto/timeseries.proto

```diff
@@ -78,8 +96,10 @@ message TimeSeriesQueryResponse {
         optional string name = 1 [(gogoproto.nullable) = false];
         // A list of sources from which the data was aggregated.
         repeated string sources = 2 [(gogoproto.nullable) = false];
+        // The aggregation function applied to points in the result.
+        optional TimeSeriesQueryAggregator aggregator = 3 [default = AVG];
```

Comment:
Maybe, but there also may be a soup of promises and callbacks on the client when they get to the results; this seems helpful.